src/oom/oomd-util.c

```diff
@@ -210,14 +212,31 @@ int oomd_cgroup_kill(const char *path, bool recurse, bool dry_run) {
 
 int oomd_kill_by_pgscan(Hashmap *h, const char *prefix, bool dry_run) {
         _cleanup_free_ OomdCGroupContext **sorted = NULL;
+        uid_t uid;
```

Comment:
uid is sometimes initialized by the loop below, and sometimes not. should be reliably initialized i guess